.github/workflows/publish-r2.yaml

```
@@ -45,4 +45,5 @@ jobs:
           BUCKET: ${{ secrets.R2_BUCKET }}
           PATH_TO_SYNC: ${{ inputs.path }}
         run: |
-          rclone -v sync --checksum --no-update-modtime $PATH_TO_SYNC :s3,env_auth:$BUCKET/
+          rclone -v sync --checksum --no-update-modtime --config="/dev/null" \
+          $PATH_TO_SYNC :s3,env_auth:$BUCKET/
```

nit: can we indent this line so it's visually obvious it's a continuation of the same command?

```suggestion
            $PATH_TO_SYNC :s3,env_auth:$BUCKET/
```